pages/browser-support.html

```diff
@@ -7,26 +7,30 @@ <h2>Current Active Support</h2>
 <h3>Desktop</h3>
 <ul>
 	<li>Chrome: (Current - 1) and Current</li>
-	<li>Edge: (Current - 1) and Current</li>
+	<li>Edge: (Current - 1) and Current<sup>[1]</sup></li>
 	<li>Firefox: (Current - 1) and Current, ESR</li>
-	<li>Internet Explorer: 9+</li>
+	<li>Internet Explorer: 11</li>
 	<li>Safari: (Current - 1) and Current</li>
 	<li>Opera: Current</li>
 </ul>
 
 <h3>Mobile</h3>
 <ul>
-	<li>Stock browser on Android 4.0+</li>
-	<li>Safari on iOS 7+</li>
+	<li>Chrome on Android: (Current - 1) and Current</li>
+	<li>Safari on iOS: (Current - 2), (Current - 1) and Current</li>
 </ul>
 
+<p>[1] Only the Chromium-based Edge is supported, the old EdgeHTML-based version is not.</p>
```

Comment:
Note: this is not decided yet, see https://github.com/jquery/jquery/issues/4568 for an ongoing discussion.

Reply:
This has been decided now.